package.json

```diff
@@ -20,6 +20,7 @@
     "build-server": "tsc --project tsconfig.server.json"
   },
   "dependencies": {
+    "dotenv": "^6.2.0",
```

Comment:
You don't need to install `dotenv` at all, as this is already built into `next`
https://nextjs.org/docs/basic-features/environment-variables